test/dotnet-test.Tests/GivenThatWeWantToRunTestsInTheConsole.cs

```diff
@@ -21,14 +21,17 @@ public GivenThatWeWantToRunTestsInTheConsole()
         {
             var testAssetManager = new TestAssetsManager(Path.Combine(RepoRoot, "TestAssets"));
             var testInstance =
-                testAssetManager.CreateTestInstance("ProjectWithTests", identifier: "ConsoleTests").WithLockFiles();
+                testAssetManager.CreateTestInstance("ProjectWithTests", identifier: "ConsoleTests");
 
             _projectFilePath = Path.Combine(testInstance.TestRoot, "project.json");
             var contexts = ProjectContext.CreateContextForEachFramework(
                 _projectFilePath,
                 null,
                 PlatformServices.Default.Runtime.GetAllCandidateRuntimeIdentifiers());
 
+            // Restore the project again in the destination to resolve projects
```

Comment:
There's no "issue" to be fixed. It's by design. Project paths are now stored in the lock file as relative paths and as a result, the project needs to be re-restored when it is moved to a different folder structure.